src/Illuminate/Foundation/Console/ShowModelCommand.php

```diff
@@ -421,6 +447,38 @@ protected function columnIsUnique($column, $indexes)
             ->contains(fn (Index $index) => $index->isUnique());
     }
 
+    /**
+     * Install the command needed dependencies.
+     *
+     * @return void
+     *
+     * @throws \Symfony\Component\Process\Exception\ProcessSignaledException
+     */
+    protected function installDependencies()
+    {
+        $command = collect($this->composer->findComposer())
+            ->push('require doctrine/dbal --dev')
```

Comment:
```suggestion
            ->push('require doctrine/dbal')
```
Other features, mainly modifying a database column require this as a non dev dependency. Installing it as a dev dependency now might break someones migrations in production later.

https://laravel.com/docs/9.x/migrations#prerequisites